src/service/resource.js

```diff
@@ -265,7 +265,7 @@ export class Resource {
       this.state_ = ResourceState.NOT_LAID_OUT;
     }
     if (opt_callback) {
```

Comment:
I don't understand, this part of code has not changed. in `resources-impl.js` it will call `request.resource.changeSize()` for resize attempts that can actually be resized.